### PR TITLE
Better windows integration

### PR DESCRIPTION
**Pull request by Ztarbox**
Trying to improve functionality for Windows. 

Including atm:
- Optional menu
- Localization support
- Taskbar flashing on new message

sadly badges aren't supported by electron on windows atm.
Any other ideas?

### CODE REVIEW

**Comment by Armaldio** [comment]
Hello, 

Thank you for opening this pull request!

We will review it shortly

As a side nide, you can support badges on windows using https://github.com/electron/electron/blob/master/docs/api/browser-window.md#winsetoverlayiconoverlay-description-windows `setOverlayIcon`

What you need to do is to have a 16x16 set of icons representing numbers (1-9, 9+) and display the image according

**Comment by Armaldio** [comment]
Also, please update your branch

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (Armaldio) [inline .vscode/launch.json]: Please, do not commit .vscode files

File: translations/i18n.js [hunk 2]
```diff
@@ -9,8 +9,10 @@ let app = electron.app;
 module.exports = i18n;
 
 function i18n() {
-  if(fs.existsSync(path.join(__dirname, app.getLocale() + '.js'))) {
-    loadedLanguage = JSON.parse(fs.readFileSync(path.join(__dirname, app.getLocale() + '.js'), 'utf8'));
+  // split locals like de-DE or en-US down to the first language code
+  let local = app.getLocale().split("-")[0];
```
Comment (Armaldio) [inline translations/i18n.js]: typo: local**e**

**Comment by Ztarbox** [comment]
I thought about this too.
But it seemed so... dirty.
Will take a further look.

**Comment by Armaldio** [comment]
Popular chat app **Franz** is using this technique for example
https://github.com/meetfranz/franz/tree/master/src/assets/images/taskbar/win32

**Comment by Ztarbox** [comment]
@Armaldio it seems to work on windows.
 Though i changed i lil bit on the code regarding Mac.
Could u test this against Mac? Or is there an easy way I can?

**Comment by Armaldio** [comment]
Please, open another PR if it's still relevant

**Comment by Ztarbox** [comment]
> Please, open another PR if it's still relevant

... no thanks. don't use it anymore

**Pull request closed without merging**